react/isItBurnt/client/src/views/searchRes.js

```diff
@@ -64,14 +64,14 @@ const SearchRes = ({recipe, recipeInfo, value}) => {
                 </Card.Text>
                 <a href={recipeInfo.sourceUrl}>Full Recipe at {recipeInfo.sourceName}</a>
                 <form onSubmit={newRecipeHandler}>
-                    {/* <div> */}
-                    <div style={{display: 'none'}}>
-                        <input type="text" name='recipeName' value={recipeName} onChange={(e) => setRecipeName(e.target.value)} />
-                        <input type="text" name='recipeImage' value={recipeImage} onChange={(e) => setRecipeImage(e.target.value)} />
-                        <input type="text" name='recipeUrl' value={recipeUrl} onChange={(e) => setRecipeUrl(e.target.value)} />
-                        <input type="text" name='recipeUrlName' value={recipeUrlName} onChange={(e) => setRecipeUrlName(e.target.value)} />
-                        <textarea type="textarea" name='recipeSummary' value={recipeSummary} onChange={(e) => setRecipeSummary(e.target.value)} />
-                        <textarea type="textarea" name='recipeInstructions' value={recipeInstructions} onChange={(e) => setRecipeInstructions(e.target.value)} />
+                    <div>
+                    {/* <div style={{display: 'none'}}> */}
+                        <input type="text" name='recipeName' value={recipeInfo?.title} onChange={(e) => setRecipeName(e.target.value)} />
+                        <input type="text" name='recipeImage' value={recipeInfo?.image} onChange={(e) => setRecipeImage(e.target.value)} />
+                        <input type="text" name='recipeUrl' value={recipeInfo?.sourceUrl} onChange={(e) => setRecipeUrl(e.target.value)} />
+                        <input type="text" name='recipeUrlName' value={recipeInfo?.sourceName} onChange={(e) => setRecipeUrlName(e.target.value)} />
+                        <textarea type="textarea" name='recipeSummary' value={recipeInfo?.summary} onChange={(e) => setRecipeSummary(e.target.value)} />
+                        <textarea type="textarea" name='recipeInstructions' value={recipeInfo?.instructions} onChange={(e) => setRecipeInstructions(e.target.value)} />
```

Comment:
React will not rerender this component if you are not using setState function, so we need the parent to rerender this area by passing in the value of recipeInfo